client/ayon_core/addon/base.py

```diff
@@ -822,6 +830,17 @@ def initialize_addons(self):
                     exc_info=True
                 )
 
+        for item in aliased_names:
+            name_alias, addon = item
```

Comment:
```suggestion
        for name_alias, addon in aliased_names:
```

Reply:
Some pythons have issue with this expanding in for loop.

Reply:
Which python? 

It works fine for me in Py 2.7, Py 3.7, Py 3.9 and Py 3.10 with the next code snippet:
```python
data = [(1, 2), (1001, 1010)]
for start, end in data:
    print(start, end)
```

Reply:
To be honest I don't remember, but I'm scared to use this now as we need to make release today.